src/app/editor-tabs/ace.component.spec.ts

```diff
@@ -103,6 +104,24 @@ describe('AceComponent', () => {
     }));
   }));
 
+  it('reloads editor content on successful save to account for concurrent, non-conflicting, automatically merged changes', fakeAsync(() => {
```

Comment:
please add a test that ensures that the cursor position is not changed (or even better, the cursor restores its position to the most appropriate position, which might be different because of remote removals in the source code).

Reply:
this went into a separate ticket